tests/core/fixtures.py

```diff
@@ -77,12 +83,70 @@ def chain(chaindb):
     return chain
 
 
+SHARD_CHAIN_CONTRACTS_FIXTURE = {
+    "contract_code": contract_bytecode,
+    "deployed_address": contract_address,
+    "initial_balance": 100000000,
+}
+
+
 @pytest.fixture
 def shard_chain():
     shard_chaindb = BaseChainDB(get_db_backend(), state_backend_class=FlatTrieBackend)
     return chain(shard_chaindb)
 
 
+@pytest.fixture
+def shard_chain_without_block_validation():
+    """
+    Return a Chain object containing just the genesis block.
+
+    This Chain does not perform any validation when importing new blocks.
+
+    The Chain's state includes one funded account which is where the simple transfer
+
+    contract will be deployed at.
+    """
+    # TODO: Comment this out to use NestedTrieBackend to prevent access list validation.
+    # Uncomment this to use FlatTrieBackend once access list generation is implemented.
+    # shard_chaindb = BaseChainDB(get_db_backend(), state_backend_class=FlatTrieBackend)
+    shard_chaindb = BaseChainDB(get_db_backend())
+    overrides = {
+        'import_block': import_block_without_validation,
+        'validate_block': lambda self, block: None,
+    }
+    klass = Chain.configure(
+        name='TestShardChainWithoutBlockValidation',
+        vm_configuration=(
+            (constants.GENESIS_BLOCK_NUMBER, ShardingVM),
+        ),
+        **overrides,
+    )
+    genesis_params = {
+        'block_number': constants.GENESIS_BLOCK_NUMBER,
+        'difficulty': constants.GENESIS_DIFFICULTY,
+        'gas_limit': constants.GENESIS_GAS_LIMIT,
+        'parent_hash': constants.GENESIS_PARENT_HASH,
+        'coinbase': constants.GENESIS_COINBASE,
+        'nonce': constants.GENESIS_NONCE,
+        'mix_hash': constants.GENESIS_MIX_HASH,
+        'extra_data': constants.GENESIS_EXTRA_DATA,
+        'timestamp': 1501851927,
+        # 'state_root': decode_hex(
+        #     '0x9d354f9b5ba851a35eced279ef377111387197581429cfcc7f744ef89a30b5d4')
```

Comment:
Reason for this being commented out?  Can it be removed?

Reply:
You are right. This can be removed.